entity/contents.cc

```diff
@@ -282,7 +282,10 @@ const IRect& TextureContents::GetSourceRect() const {
  ******* SolidStrokeContents
  ******************************************************************************/
 
-SolidStrokeContents::SolidStrokeContents() = default;
+SolidStrokeContents::SolidStrokeContents() {
+  SetStrokeCap(Cap::kButt);
+  SetStrokeJoin(Join::kBevel);
```

Comment:
For Skia and Flutter, the default is `kMiter`.

Reply:
I see miter isn't implemented yet, we should have a TODO here at least to default this to kMiter.

Reply:
Yeah, but I need to set it to kBevel by default until kMiter is implemented (coming in a few mins).

Reply:
Done.